pkg/gpu/probe.go

```diff
@@ -46,7 +46,7 @@ type ProbeDependencies struct {
 
 // Probe represents the GPU monitoring probe
 type Probe struct {
-	mgr            *ddebpf.Manager
+	m              *manager.Manager
```

Comment:
Why are we switching to use the unwrapped manager? We'll need it if we want to add telemetry to the eBPF probes, no?